src/snackbar/index.unit.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import Snackbar from 'snackbar'
+import Button from 'button'
+import exenv from 'exenv'
+
+const close = jest.fn()
+
+describe('Snackbar', () => {
+  it('should show the snackbar when isOpen is true', () => {
+    const wrapper = mount(<Snackbar isOpen close={close} />)
```

Comment:
Remove the usage of `mount` in the tests to use shallow instead  when you can

Reply:
Done